kivymd/uix/button/button.py

```diff
@@ -1109,9 +1116,16 @@ class MDIconButton(ButtonContentsIcon, BaseButton):
     """
     Button icon.
 
-    :attr:`icon` is an :class:`~kivy.properties.StringProperty`
+    :attr:`icon` is a :class:`~kivy.properties.StringProperty`
     and defaults to `'checkbox-blank-circle'`.
     """
+
+    # Deprecated for MDIcon - use icon_color and theme_icon_color
+    text_color = ColorProperty(None, deprecated=True)
+    theme_text_color = OptionProperty(
+        None, options=theme_text_color_options, deprecated=True
+    )
+
```

Comment:
text_color = ColorProperty(None, deprecated=True)
    """
    Description
    ...
    """

    theme_text_color = OptionProperty(
        None, options=theme_text_color_options, deprecated=True
    )

Reply:
Better details of deprecation given for MDIconButton properties text_color and theme_text_color.